README.md

```diff
@@ -22,148 +19,183 @@ Table of contents
+  `./src/offlineData` directory.  
+4. In the `.src/utils/Mock.js` file, add the following variable for data:
+  `const myData = require('../offlineData/myData.json')`
+5. Add data into the `mockData.set(myData.self.uri, { status: myStatusCode, data: myData}` map.
+    * For **followlocation**, create a new variable for the request uri and use that variable instead of the `myData.self.uri` variable. The responses include the *followed* url instead of the *request* url.
+    * For a request with no response, add the request URL with a status code and empty data. For more information, see the forms in the *Mock.js* file.
+
+### Verifying Data
+
+After mocking data for *flow*, enable [Offline Mode](#configuration) mode, restart the server and verify that the storefront is functioning as required.<br/>
+Go through your *flow* and verify everything works the same as Online. For any missing response, an error is displayed in the *browser console*, which includes the request for which mock data is not found. Mock that request to continue.
+
+### Default flows
+By default, mock data for the following flows are available:
+1. In the **Search** filed, enter `water` and view the products returned.
+2. Navigate to the **Women** category.<br/>The products in the category are displayed.
+3. In the **Women** category, add *Women's CR550 Polo"* to cart.<br/>**Note** Currently, only grey in medium size is available.
+4. Check out the cart. There's currently one product in there for viewing.<br/>**Note** You cannot modify the cart. The `./src/offlineData/cartData.json` response is static.
+5. In the cart, click  *Proceed To Checkout*.<br/> Currently, you can continue only without an account.
+6. On the checkout page, complete the order by using the mock data for billing address, shipping address, and payment method.
+    * When the order is complete, the **Order Review** page is displayed.
+7. To view the order information, complete the purchase.
+     The **Purchase Receipt** page with the order information is displayed.
+
+## Adding New Locales
+
+The reference storefront supports multiple languages and currencies. Add all front-end textual data in the `localization/en-CA.json` file as a resource string.By default,  `en-CA` and `fr-FR` are provided in this project.
+
+1. For development purpose, run: `node tools/translate.js`.<br/>This runs a pseudo translation from `en-CA` locale to `fr-FR`.
+2. To add a new locale, add an entry to the `supportedLocales` array in `ep.config.json` file and add an appropriate `.json` file to the `localization` folder.
+3. Configure the language and currency for all products in the Commerce Manager.
+
+## Running Unit Tests
+
+Test data are provided in the `tests` directory.
+
+1. To run all tests, run the following command:<br/> `mvn clean install -Dcucumber.options="--tags @smoketest"`<br/>
+2. To run sanity t, run the following command:<br/>`@sanity`<br/>
```

Comment:
Should be:
"To run sanity tests, run the following command `mvn clean install -Dcucumber.options="--tags @smoketest" `"